assembl/lib/sqla.py

```diff
@@ -848,6 +853,60 @@ def translate_to_json(v):
 
     dummy_context = DummyContext()
 
+    def locked_object_creation(
```

Comment:
This method works great for the use-case that it was made for. However, perhaps a comment or note should be placed on its limitation, @maparent . That way it won't be used a general purpose locking mechanism under all conditions by unknowing devs. Agree?